internal/backend/sessions/sessionworkflows/metrics.go

```diff
@@ -0,0 +1,35 @@
+package sessionworkflows
+
+import (
+	"context"
+	"fmt"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/telemetry"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+	"go.opentelemetry.io/otel/metric"
+)
+
+var sessionsCounter metric.Int64UpDownCounter
+
+func initMetrics(t *telemetry.Telemetry) {
+	sessionsCounter = t.NewOtelUpDownCounter("sessions.gaude", "The session counter")
+}
+
+func updateSessionCounter(sessionID string, sessionState sdktypes.SessionStateType) {
+	labels := telemetry.Labels{"session_id": sessionID}
```

Comment:
sessions_id is ultra high cardinality dimension, and it has no use in metrics

you would never use any arithmetic calculation based on session_id

Reply:
agree. removed